### PR TITLE
Get information on famous tv-series

### PR DESCRIPTION
**Pull request by saurabhraj1896**
please review this , @casey-collab

### CODE REVIEW

File: english/famous-tv-series/Sherlock-Holmes/imdb/imdb.md [hunk 1]
```diff
@@ -0,0 +1 @@
+[Click here](http://www.imdb.com/title/tt1475582/)
```
Comment (casey-collab) [inline english/famous-tv-series/Sherlock-Holmes/imdb/imdb.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

File: english/famous-tv-series/famous-tv-series.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+List of famous TV series:
```
Comment (casey-collab) [inline english/famous-tv-series/famous-tv-series.md]: This page is also unreachable.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by saurabhraj1896** [comment]
please again review, @casey-collab

File: english/famous-tv-series/famous-tv-series.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+ 1) Game-of-thrones [click for more-info](Game-of-thrones/Game-of-thrones.md)
```
Comment (casey-collab) [inline english/famous-tv-series/famous-tv-series.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: english/famous-tv-series/famous-tv-series.md [hunk 4]
```diff
@@ -0,0 +1,3 @@
+ 1) Game of thrones [click for more-info](Game-of-thrones/Game-of-thrones.md)
```
Comment (saurabhraj1896) [inline english/famous-tv-series/famous-tv-series.md]: please review this, @casey-collab

**Comment by saurabhraj1896** [comment]
please review, @casey-collab

File: english/famous-tv-series/famous-tv-series.md (hunk 4 above)
Comment (casey-collab) [inline english/famous-tv-series/famous-tv-series.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by saurabhraj1896** [comment]
please review, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :star:

**Comment by saurabhraj1896** [comment]
thanks for merging @casey-collab

**Comment by casey-collab** [comment]
Unfortunately, this pull request conflicts with another change. Could you merge the master branch of the original repository into your branch, resolve the merge conflicts, and update the pull request?

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by casey-collab** [comment]
Unfortunately, this pull request conflicts with another change. Could you merge the master branch of the original repository into your branch, resolve the merge conflicts, and update the pull request?

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Pull request merged**